modules/scrollback/index.js

```diff
@@ -110,108 +111,389 @@ module.exports.formatLine = function(line) {
   }
 };
 
+/* Abstract: Quoting lines from IRC is an essential problem with far-reaching
+ * impact: many humans and semi-sentient AIs depend on their ability to precisely
+ * and concisely specify exactly which embarrasing statements and/or nonsequitirs
+ * presented in a real-time group chat system are worthy of being selected for
+ * processing by other software, including various quotation recording libraries,
+ * machine learning algorithms, and text processing utilities. In this function,
+ * the authors present a novel approach to the formatted text selection problem:
```

Comment:
I'm going to get funding for IRC bot research on EuIrcBot; this shit is going to be so theoretically applicable!